src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write code here
+const container = document.querySelector('.container');
+const field = container.querySelector('.field tbody');
+
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+container.addEventListener('click', e => {
+  const button = e.target.closest('.button');
+
+  if (!button) {
+    return;
+  }
+
+  switch (button.classList[0]) {
+    case 'append-row':
```

Comment:
```suggestion
    case appendRow:
```
У тебя в этой переменной записана ссылка на нужную кнопку, и она же будет в event.target, если кликнуть по кнопке

Reply:
точно, спасибо)